oraclexe/README.md

```diff
@@ -0,0 +1,25 @@
+Oracle XE Exhibit
+==============
+
+# How to Use
+
+## Download Oracle JDBC Driver
+
+[Download the Oracle JDBC driver](https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html) and place in `lib/` .
```

Comment:
There's quite a few downloads on this page.  Should we be more specific or "recommend" one?

Reply:
The `ojdbc8.jar` one worked for me, so technical part is approved.

Reply:
yeah probably just ojdbc8 for java 8 compat